nautobot/core/views/generic.py

```diff
@@ -1041,7 +1041,8 @@ def post(self, request, **kwargs):
                 try:
                     with deferred_change_logging_for_bulk_operation():
                         updated_objects = []
-                        for obj in self.queryset.filter(pk__in=form.cleaned_data["pk"]):
+                        queryset = queryset if edit_all else self.queryset.filter(pk__in=form.cleaned_data["pk"])
```

Comment:
In the non-edit-all case, doesn't this overwrite the assignment at line 1024? Should it perhaps be:

```suggestion
                        queryset = queryset if edit_all else queryset.filter(pk__in=form.cleaned_data["pk"])
```

Or alternately, is the assignment at line 1024 unnecessary/irrelevant?

Reply:
`queryset` is utilised on line 1140